src/window.py

```diff
@@ -173,13 +174,13 @@ def _update_success_msg(self, download_dir_abs):
         if os.path.commonpath([home_dir, label_dir]) == home_dir:
             label_dir = '~' + label_dir[len(home_dir):]
         template = GObject.markup_escape_text(N_('Saved in {}'))
-        link = '<a href="action:open-download-dir">{}</a>'.format(
+        link = '{}'.format(
             GObject.markup_escape_text(label_dir))
         self.success_msg_wdg.set_markup(template.format(link))
```

Comment:
It would be nice to set the directory path as a tooltip of the ``open_dir_wdg`` button. ``success_msg_wdg`` should be removed in favour of the "Download finished" label.